### PR TITLE
Print to stderr when there is an exception

### PR DESCRIPTION
**Pull request by JoseBritto**
This PR should add logs in missed places.

### CODE REVIEW

**Comment by JoseBritto** [comment]
Now this is just to make debugging for #762 a tiny bit easier. Will go  through and complete the PR by tuesday :)

**Comment by nlogozzo** [comment]
Might be best to Ctrl+Shift+F for `catch` and add and print the exception where needed...

**Comment by JoseBritto** [comment]
> Might be best to Ctrl+Shift+F for catch and add and print the exception where needed...
Alright! Also probably better to use Console.Error. I forgot that earlier

**Comment by nlogozzo** [comment]
Console.WriteLine is probably fine...stdput and stderr point to the same terminal unless the programmer usually redirects it to a separate file which we don't really in this case...

**Comment by JoseBritto** [comment]
> Console.WriteLine is probably fine...stdput and stderr point to the same terminal unless the programmer usually redirects it to a separate file which we don't really in this case...

It is actually two different streams but most terminals won't differentiate by default. Its standard thing for most programs to print use the correct output stream. Also users can manually redirect errors as well to a file or whatever. Since Denaro isn't a command line app, it probably doesn't really matter. Since I'm gonna have to go through everything anyway, might as well fix the few logs already being printed out.

**Comment by JoseBritto** [comment]
I'll complete this in the afternoon. Have been a bit busy

**Comment by JoseBritto** [comment]
Added Console.Error.WriteLine to most catch blocks that made sense to me. I ignored the ones that executed sql queries and such.

File: NickvisionMoney.GNOME/Views/AccountView.cs [hunk 1]
```diff
@@ -828,6 +834,7 @@ private async Task ExportToPDFAsync(ExportMode exportMode)
             {
                 path += ".pdf";
             }
+
```
Comment (nlogozzo) [inline NickvisionMoney.GNOME/Views/AccountView.cs:837]: Can you remove these extra white spaces?

File: NickvisionMoney.GNOME/Views/AccountView.cs [hunk 2]
```diff
@@ -849,11 +856,15 @@ private async Task ExportToPDFAsync(ExportMode exportMode)
                 {
                     _controller.ExportToPDF(path ?? "", exportMode, null);
                 }
+
```
Comment (nlogozzo) [inline NickvisionMoney.GNOME/Views/AccountView.cs:859]: Here too

File: NickvisionMoney.GNOME/Views/NewAccountDialog.cs [hunk 3]
```diff
@@ -269,14 +269,18 @@ private async void SelectFolder(object? sender, EventArgs e)
             var folder = Gio.FileHelper.NewForPath(_controller.Folder);
             folderDialog.SetInitialFolder(folder);
         }
+
```
Comment (nlogozzo) [inline NickvisionMoney.GNOME/Views/NewAccountDialog.cs:272]: Here too

**Review by nlogozzo — CHANGES_REQUESTED** [review]

**Comment by JoseBritto** [comment]
Idk why rider keeps adding these in

**Review by nlogozzo — APPROVED** [review]
Thanks!

**Comment by nlogozzo** [comment]
https://github.com/NickvisionApps/Denaro/releases/tag/2024.1.0

**Pull request merged**